anybox/recipe/odoo/runtime/session.py

```diff
@@ -191,7 +190,7 @@ def init_environments(self):
         See :class:``openerp.api.Environment`` for explanations about
         environments.
 
-        For OpenERP/Odoo versions prior to the new style API merge, this
+        For Odoo/Odoo versions prior to the new style API merge, this
```

Comment:
We have to adjust this comment! Otherwise I think we should stop to maintain Version 7 in at some point (as we have anybox.recipe.openerp for that) but let do that in an other PR ;)